submissions/LuckyDnepr/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,226 @@
+                  </li>
+                  <li class="nav_popup_menu_item">
+                    <a href="#Chat" class="nav_popup_menu_link">
+                      <img
+                        class="nav_popup_menu_icons"
+                        src="./icons/chat.png"
+                        title="Chat"
+                        alt="Chat"
+                      />
+                    </a>
+                  </li>
+                  <li class="nav_popup_menu_item">
+                    <a href="#Classroom" class="nav_popup_menu_link">
+                      <img
+                        class="nav_popup_menu_icons"
+                        src="./icons/classroom.png"
+                        title="Classroom"
+                        alt="Classroom"
+                      />
+                    </a>
+                  </li>
+                  <li class="nav_popup_menu_item">
+                    <a href="#Contacts" class="nav_popup_menu_link">
+                      <img
+                        class="nav_popup_menu_icons"
+                        src="./icons/contacts.png"
+                        title="Contacts"
+                        alt="Contacts"
+                      />
+                    </a>
+                  </li>
+                  <li class="nav_popup_menu_item">
+                    <a href="#Documents" class="nav_popup_menu_link">
+                      <img
+                        class="nav_popup_menu_icons"
+                        src="./icons/documents.png"
+                        title="Documents"
+                        alt="Documents"
+                      />
+                    </a>
+                  </li>
+                  <li class="nav_popup_menu_item">
+                    <a href="#Files" class="nav_popup_menu_link">
+                      <img
+                        class="nav_popup_menu_icons"
+                        src="./icons/files.png"
+                        title="Files"
+                        alt="Files"
+                      />
+                    </a>
+                  </li>
+                  <li class="nav_popup_menu_item">
+                    <a href="#Maps" class="nav_popup_menu_link">
+                      <img
+                        class="nav_popup_menu_icons"
+                        src="./icons/maps.png"
+                        title="Maps"
+                        alt="Maps"
+                      />
+                    </a>
+                  </li>
+                </ul>
+                <input id="popup_menu_show_more_check" type="checkbox" />
+                <label
+                  id="popup_menu_show_more_label"
+                  for="popup_menu_show_more_check"
+                >
+                  <span class="popup_menu_show_more show_more_span"
+                    >Show more</span
+                  >
+                  <span class="popup_menu_show_more show_less_span"
+                    >Show less</span
+                  >
```

Comment:
I'm not sure about this solution: How does it look like from a logical perspective: button "show less"  in the middle of popup.

Reply:
I can't make the right decision. 
If I understand correctly how the checkbox works in this situation, there are two options:
1) we place the checkbox at the bottom of the popup (via flexbox), but for correctly work, therefor the checkbox in HTML it must stand before the section it opens.
In this case on <tab>, after menu we will go to <show more/less> button, which seems not very logical.
2) Or after pressing <show more> hide this button completely.

Reply:
Abstract example: From UX perspective we have a list of items and we show some parts of this list, when users want to download more items they click button "download more" and items add to the end of the list but before button "download more".
About the current task, we have one main restriction: no JS. Without JS we can't do this correctly, so you must implement a show more button with hides after showing the second part of the popup.

Reply:
I resolve it. Please review.